src/components/Button/Button.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from 'react';
+
+import classNames from 'classnames';
+
+import { Icon } from '../Icon/Icon';
+
+export interface ButtonProps {
+	label?: string;
```

Comment:
none are required:
![image](https://user-images.githubusercontent.com/1710840/58540985-c47f6080-81fa-11e9-96ce-167ba4c06a8f.png)

Reply:
Klopt, da's omdat dat ook zo is bij de props.